package.json

```diff
@@ -31,11 +31,9 @@
     "leven": "^3.1.0"
   },
   "dependencies": {
+    "@canvas/image": "^1.0.0",
     "blockhash": "^0.2.0",
-    "bmp-js": "0.1.0",
-    "image-type": "^2.1.0",
     "jpeg-js": "^0.3.6",
-    "png-js": "^0.1.1",
-    "pngjs": "^3.4.0"
+    "png-js": "^0.1.1"
```

Comment:
`blockhash` tries to import `png-js` and `jpeg-js` without having it's own dependency on them, so for now I kept them here. This should really be fixed upstream though!

Reply:
True, true... I stumbled upon it here https://github.com/pwlmaciejewski/imghash/issues/5#issuecomment-548101799 but apparently I forgot to create an issue for it. There: https://github.com/pwlmaciejewski/imghash/issues/17